internal/tweets/classifier.go

```diff
@@ -3,32 +3,49 @@ package tweets
 import (
 	"context"
 	"fmt"
-
 	"github.com/kordape/tweety/internal/entity"
+	ml_model "github.com/kordape/tweety/internal/tweets/ml-model"
+	"github.com/kordape/tweety/internal/tweets/webapi"
 )
 
 type Classifier struct {
-	webAPI TwitterWebAPI
+	webAPI  TwitterWebAPI
+	mlModel MLModel
 }
 
-func NewClassfier(w TwitterWebAPI) *Classifier {
+func NewClassifier(w TwitterWebAPI, m MLModel) *Classifier {
 	return &Classifier{
-		webAPI: w,
+		webAPI:  w,
+		mlModel: m,
 	}
 }
 
 // Classify - classifies if tweets are fake news
-func (classifier *Classifier) Classify(ctx context.Context, pageId string) ([]entity.TweetWithClassification, error) {
-	tweets, err := classifier.webAPI.FetchTweets(ctx, pageId)
+func (classifier *Classifier) Classify(ctx context.Context, ftr webapi.FetchTweetsRequest) ([]entity.TweetWithClassification, error) {
```

Comment:
Ne bi trebao da koristis request iz `webapi` paketa ovde. Meni je bilo ok ono sto si u prosloj iteraciji napravio (isti taj request samo u ovom paketu), sto si pomerio to?

Verovatno ti trebaju oba u stvari:
- jedan request za classifier
- jedan za twitter api

tako je najcistije i najvise ima smisla, i time ne pravis nikakav dependency